docs/configuration.rst

```diff
@@ -5,222 +5,47 @@ Configuration
-
-    {
-        "description": "GRCh37 primary assembly",
-        "sourceUri": "TODO",
-        "assemblyId": "TODO",
-        "sourceAccessions": [],
-        "isDerived": false,
-        "ncbiTaxonId": 9606
-    }
-
-Within a reference set directory is a set of files defining the references
-themselves. Each reference object corresponds to three files: the bgzip
-compressed FASTA sequences, the FAI index and a JSON file providing the
-metadata. There must be exactly one sequence per FASTA file, and the
-sequence ID in the FASTA file must be equal to the reference name
-(i.e., the first line in ``1.fa`` above should start with ``>1``.)
-
-The JSON metadata required for a reference is similar to a reference set.
-An example might look something like:
-
-.. code-block:: json
-
-    {
-        "sourceUri": "TODO",
-        "sourceAccessions": [
-            "CM000663.2"
-        ],
-        "sourceDivergence": null,
-        "md5checksum": "bb07c91cda4645ad8e75e375e3d6e5eb",
-        "isDerived": false,
-        "ncbiTaxonId": 9606
-    }
-
-
-++++++++++
-Datasets
-++++++++++
-
-The main container for genetic data is the dataset. Within the
-main data directory there must be a directory called ``datasets``.
-Within this directory each subdirectory is interpreted as a
-dataset of that name. For example, we might have something like::
-
-    datasets/
-        1kg-phase1
-            variants/
-                # Variant data
-            reads/
-                # Read data
-        1kg-phase3
-            variants/
-                # Variant data
-            reads/
-                # Read data
-
-In this case we specify two datasets with name equal to ``1kg-phase1`` and
-``1kg-phase3``. These directories contain the read and variant data
-within the ``variants`` and ``reads`` directory, respectively.
-
-++++++++
-Variants
-++++++++
-
-Each dataset can contain a number of VariantSets, each of which basically
-corresponds to a VCF file. Because VCF files are commonly split by chromosome
-a VariantSet can consist of many VCF files that have consistent metadata.
-Within the ``variants`` directory, each directory is interpreted as a
-variant set with that name. A variant set directory then contains
-one or more indexed VCF/BCF files.
-
-+++++
-Reads
-+++++
-
-A dataset can contain many ReadGroupSets, and each ReadGroupSet contains
-a number of ReadGroups. The ``reads`` directory contains a number of BAM
-files, each of which corresponds to a single ReadGroupSet. ReadGroups are
-then mapped to the ReadGroups that we find within the BAM file.
+The repository in the GA4GH reference server defines how your data is organised. The
+repository itself is a SQLite database, which contains information about your
+datasets, reference sets and so on. Bulk data (such as variants and reads)
+is not stored in database, but instead accessed directly from the primary
+data files at run time. The locations of these data files is entirely up
+to the administrator.
```

Comment:
We would need to be precise about how and when here. It would just be confusing to say that we'll sometimes need write access without fully qualifying.